src/components/Table/Table.vue

```diff
@@ -131,12 +135,20 @@ const sort = (field: Field) => {
     const sort = sorted[key];
 
     if (field.sortable) {
-        if (sort === 'asc') {
-            sorted[key] = 'desc';
-        } else if (sort === 'desc') {
-            delete sorted[key];
+        if (props.multiSort) {
+            if (sort === 'asc') {
+                sorted[key] = 'desc';
+            } else if (sort === 'desc') {
+                delete sorted[key];
+            } else {
+                sorted[key] = 'asc';
+            }
         } else {
-            sorted[key] = 'asc';
+            for (const member in sorted) {
+                delete sorted[member];
+            }
+
+            sorted[key] = sort === 'asc' ? 'desc' : 'asc';
```

Comment:
To prevent unnecessary triggers for each of the changes.
```suggestion
            sorted.value = {
                [key]: sort === 'asc' ? 'desc' : 'asc',
            };
```
However this is currently not possible since `sorted` is not a `ref` but `reactive`. Personally I would go with `ref`.